ibis/backends/tests/sql/snapshots/test_compiler/test_subquery_where_location/decompiled.py

```diff
@@ -11,7 +11,7 @@
     },
 )
 param = ibis.param("timestamp")
-f = alltypes.filter((alltypes.timestamp_col < param.name("my_param")))
+f = alltypes.filter((alltypes.timestamp_col < param))
```

Comment:
I modified this test to avoid the name since it didn't effectively matter. Roundtripping an aliased parameter used in a context where aliases are ignored doesn't seem necessary.